movidius_ncs_stream/tests/unittest_stream.cpp

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2017 Intel Corporation. All Rights Reserved
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <opencv2/highgui/highgui.hpp>
+
```

Comment:
Please pay attention to the order of included header files, and remove empty lines between includes.